### PR TITLE
New examples guidelines

### PR DESCRIPTION
**Pull request by FadySamirSadek**
solves #1886

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -22,6 +22,13 @@ Please bear in mind that your project should have reached a certain level of
 maturity before we can include it. If you haven't settled on a relatively stable
 API, you should probably wait a bit longer before submitting your example.
 
+### A good app that we are looking for should check most if not all of the items below :
+
+- [x] The framework/stack you are using must have a version that is free and open source
+- [x] The framework/stack you are using is at least 1 year old.
+- [x] The framework you are using should have at least 5k stars if it is hosted on Github or equivalent on other platforms.
+- [x] Your app should be able to work on recent versions of all major browsers.
+- [x] Your app should bring something truly new to the table, but minor variations of existing frameworks , languages or existing TodoMVC apps.
```
Comment (Ahmed-90) [inline contributing.md]: I believe that's a typo? "~~but~~ **not** minor variations..."?

Comment (FadySamirSadek) [reply contributing.md]: Will fix it, thanks 👍

**Comment by FadySamirSadek** [comment]
@whitefire0 I do not think an exhaustive list of features is the solution I think the first four guidelines are yes-no questions which is what we need but the last one just means your example has to be different because people can always use the same stack in different ways. I think we will always have to discuss and think if an app is a good addition or not but the first 4 guidelines  are going to help us a lot in the decision making process

**Comment by FadySamirSadek** [comment]
@Ahmed-90 Fixed the typo and rephrased the statement.
 can you please have a look? Thanks 🙂

**Comment by Ahmed-90** [comment]
@FadySamirSadek Looks good to me. 👍 
BTW I still think that this list should be enhanced with a technical (lighthouse like) checklist to ensure standardization.

**Comment by FadySamirSadek** [comment]
@Ahmed-90 I think this should be added incrementally. I think we should add lighthouse to our CI system but this can be added in an hour in the future. Will open an issue for it and feel free to submit a PR.

**Pull request merged**